src/labels/CustomLabels.labels

```diff
@@ -4214,6 +4270,548 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
+    </labels>
+    <labels>
+        <fullName>geBodyBatchHeaderRightCol</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Text block about setting up batch header fields</shortDescription>
+        <value>For each field, you can edit the field label, mark it as required, or assign a default value. Default values can be overwritten by the person who creates the batch.</value>
+    </labels>
+    <labels>
+        <fullName>geBodyBatchHeaderWarning</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Warning body of required field added to Batch Header</shortDescription>
+        <!-- {0} is comma separated list of field api names from the Data Import Batch object -->
+        <!-- e.g. npsp__GiftBatch__c, npsp__Batch_Description__c -->
+        <value>These required fields were automatically added to the Batch Header:</value>
+    </labels>
+    <labels>
+        <fullName>geBodyFormFieldsLeftCol</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Template Builder Form Fields tab left column body</shortDescription>
+        <value>Choose fields to include in your form. Before adding fields, click the section in the main screen where you want the fields to go. In the sidebar, select the checkbox to add a field, and deselect it to remove the field from the form. If the object or field you want to use isn't listed, make sure you've set up the field with Advanced Mapping.</value>
+    </labels>
+    <labels>
+        <fullName>geBodyFormFieldsLeftColAdditional</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>More information about adding and mapping fields</shortDescription>
+        <value>
+        Set up a field by completing the following steps:
+        &lt;/br&gt;
+        &lt;/br&gt;
+        &lt;ol&gt;
+            &lt;li&gt; Create the field on the target object. &lt;/li&gt;
+            &lt;li&gt; Create the field on the NPSP Data Import object. &lt;/li&gt;
+            &lt;li&gt; If this is part of a new Object Group, create additional fields for a new Object group and create the new Object Group in NPSP Settings. &lt;/li&gt;
+            &lt;li&gt; Add Field Mapping in NPSP Settings to connect the NPSP Data Import field to the target field. &lt;/li&gt;
+        &lt;/ol&gt;
+        &lt;/br&gt;
+        See NPSP documentation about Advanced Mapping for additional details.
+        </value>
+    </labels>
+    <labels>
+        <fullName>geBodyFormFieldsLeftColReadMore</fullName>
```

Comment:
@psadie-sfdo Should this label be more generalized or should we keep it specific to the instance when we use "Read more" here?

Reply:
@melissabarber - I think that makes sense. Updating this.